convert.py

```diff
@@ -58,7 +58,7 @@
     for dev, direction, string in protocol:
         print("%s %s%s" % (dev, "> " if direction == "0" else "  < ", repr(string)[1:-1]))
 
-elif mode == "extract":
+elif mode == "dump":
```

Comment:
We should leave the terminology at "dump / flash". It's a somewhat established standard.

Reply:
Yeah, I figured.

The `convert.py` script has “usage” documentation for a mode called `dump`, but a part of your implementation in fact expects `extract`. Clearly a typo. This change fixes that in favour of `dump`.

(Not sure if perhaps `convert.py` is no longer necessary and should be removed entirely under #14.)